lib/spack/llnl/util/cpu/microarchitectures.json

```
@@ -1239,17 +1252,30 @@
           },
           {
             "versions": "7:7.9",
-            "flags": "-arch=armv8.2a+crc+crypt+fp16"
+            "flags": "-arch=armv8.2a+crc+crypto+fp16"
```

Fixes `-arch` to `-march` is deleted. Please fix typo again.
```suggestion
            "flags": "-march=armv8.2a+crc+crypto+fp16"
```